### ISSUE TITLE
calendar is not accessible via screenreader

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#616]
There are a couple of a11y issues with the calendar:
- Dates do not behave as buttons for a screen reader — the `role="button"` attribute should solve that
- Dates do not convey their state via screen reader — the `aria-state="[true/false/mixed]"` attribute will probably be the right way to solve this, along with a screen-reader-accessible explanation that "mixed" state means an unselected but not blacked out day (and false would indicate a blacked out day).

**Comment by bmathews** [comment]
This might be worth checking out: http://pa11y.org/
